astropy/io/fits/diff.py

```diff
@@ -29,7 +29,7 @@
 from .card import Card, BLANK_CARD
 from .header import Header
 # HDUList is used in one of the doctests
-from .hdu.hdulist import fitsopen, HDUList  # pylint: disable=W0611
```

Comment:
Right, I guess I meant I'm surprised that it does take into account the module locals since it is by no means guaranteed the user will also have these in the namespace. But I guess the point is to test the functions rather than make sure the user can always reproduce the results without issues.